docs/docs/personalization.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Personalization
+---
+
+## What is Personalization?
+
+- Have you ever landed on a website that greeted you by name?
+
+- Did you ever search for a book and realize the suggested results were based on your previous purchases?
+
+These are examples of personalization. Personalization adjusts the content of a website, and creates a custom experience for users. With personalization, one website can offer a variety of experiences to different visitors.
+
+## Why is Personalization Important?
+
+If you are [creating an e-commerce site](/tutorial/ecommerce-tutorial/), personalization is key in converting your visitors into buyers. Online merchants use personalization as a way to suggest relevant items to a shopper.
```

Comment:
I'd recommend sending them to the e-commerce section, as there is more than one approach and the tutorial focuses on Stripe only.
```suggestion
If you are [creating an e-commerce site](/docs/building-an-e-commerce-site/), personalization is key in converting your visitors into buyers. For example, online merchants use personalization as a way to suggest relevant items to a shopper.
```